pkg/repositories/repositories.go

```diff
@@ -0,0 +1,8 @@
+package repositories
+
+type Repositories struct {
```

Comment:
Convenience structure to keep all repositories in one spot and pass them around easily to the services.